dev/devicelab/manifest.yaml

```diff
@@ -564,7 +564,7 @@ tasks:
       Measures the runtime performance of animations after a backdrop filter is removed on iOS.
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios"]
-    flaky: true
+    flaky: false
```

Comment:
It's cleaner to remove the `flaky: ...` line completely to mark it as nonflaky.

Reply:
Done